sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_logical_subscriptions_history_v1/query.sql

```diff
@@ -109,83 +137,107 @@ SELECT
   history.valid_from,
   history.valid_to,
   history.id AS provider_subscriptions_history_id,
-  STRUCT(
-    CONCAT(
-      'Stripe-',
-      history.subscription.id,
-      '-',
-      FORMAT_TIMESTAMP('%FT%H:%M:%E6S', history.subscription_first_active_at)
-    ) AS id,
-    'Stripe' AS provider,
-    IF(paypal_subscriptions.subscription_id IS NOT NULL, 'PayPal', 'Stripe') AS payment_provider,
-    history.subscription.id AS provider_subscription_id,
-    subscription_item.id AS provider_subscription_item_id,
-    history.subscription.created AS provider_subscription_created_at,
-    history.subscription.customer.id AS provider_customer_id,
-    history.subscription.customer.metadata.userid AS mozilla_account_id,
-    history.subscription.customer.metadata.userid_sha256 AS mozilla_account_id_sha256,
-    CASE
-      -- Use the same address hierarchy as Stripe Tax after we enabled Stripe Tax (FXA-5457).
-      -- https://stripe.com/docs/tax/customer-locations#address-hierarchy
-      WHEN DATE(history.valid_to) >= '2022-12-01'
-        AND (
-          DATE(history.subscription.ended_at) >= '2022-12-01'
-          OR history.subscription.ended_at IS NULL
-        )
-        THEN COALESCE(
-            NULLIF(history.subscription.customer.shipping.address.country, ''),
-            NULLIF(history.subscription.customer.address.country, ''),
-            charge_summaries.latest_card_country
-          )
-      -- SubPlat copies the PayPal billing agreement country to the customer's address.
-      WHEN paypal_subscriptions.subscription_id IS NOT NULL
-        THEN NULLIF(history.subscription.customer.address.country, '')
-      ELSE charge_summaries.latest_card_country
-    END AS country_code,
-    plan_services.services,
-    subscription_item.plan.product.id AS provider_product_id,
-    subscription_item.plan.product.name AS product_name,
-    subscription_item.plan.id AS provider_plan_id,
-    subscription_item.plan.`interval` AS plan_interval_type,
-    subscription_item.plan.interval_count AS plan_interval_count,
-    UPPER(subscription_item.plan.currency) AS plan_currency,
-    (CAST(subscription_item.plan.amount AS DECIMAL) / 100) AS plan_amount,
-    IF(ARRAY_LENGTH(plan_services.services) > 1, TRUE, FALSE) AS is_bundle,
-    IF(
-      history.subscription.status = 'trialing'
-      OR (
-        history.subscription.ended_at
-        BETWEEN history.subscription.trial_start
-        AND history.subscription.trial_end
-      ),
-      TRUE,
-      FALSE
-    ) AS is_trial,
-    history.subscription_is_active AS is_active,
-    history.subscription.status AS provider_status,
-    history.subscription_first_active_at AS started_at,
-    history.subscription.ended_at,
+  (
```

Comment:
This is mostly whitespace so using the hide whitespace option will make this easier to read